packages/workshop-app/app/utils/diff.server.ts

```diff
@@ -14,10 +14,19 @@ const kcdshopTempDir = path.join(os.tmpdir(), 'kcdshop')
 const diffTmpDir = path.join(kcdshopTempDir, 'diff')
 
 function diffPathToRelative(filePath: string) {
+	filePath = filePath.replace(/\\\\/g, path.sep).replace(/\\|\//g, path.sep)
+
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	const [workshopRootDirname, appId, ...relativePath] = filePath
-		.replace(`${diffTmpDir.slice(1)}${path.sep}`, '')
+		.replace(
+			process.platform === 'win32'
+				? `${diffTmpDir}${path.sep}`
+				: `${diffTmpDir.slice(1)}${path.sep}`,
+			'',
+		)
+		.replace(/'|"/g, '')
```

Comment:
If this only has an incorrect quote at the start of the string, then let's just remove that one. I wouldn't want to remove all quotes in the string because it's possible those quotes are intentional.

Reply:
see 13729b0